docs/content/porter-or-duffle.md

```diff
@@ -95,23 +95,18 @@ install:
           key: mysql-password
   - helm:
       description: "Install Wordpress"
-      name:
-        source: bundle.parameters.wordpress-name
+      name: {{ bundle.parameters.wordpress-name }}
```

Comment:
Seems like all the markdown files are missing quotes around the moustash template

Reply:
Yeah, I forgot to update them! Thanks for pointing that out.